README.md

```diff
@@ -50,9 +49,12 @@ The above metrics are available for Binary, Multiclass, and Multilabel classific
   - `RetrievalMAP`: computes [Mean Average Precision@k](https://stackoverflow.com/questions/54966320/mapk-computation), an information retrieval metric.
   - `RetrievalPrecisionRecallCurveNPoints`: computes a Precision-Recall Curve, interpolated at k points and averaged over all samples. 
 
-### Grounding
+### Grounding:
   - `Recall`: computes Recall@k, which is the percentage of correct grounding in top-k among all relevant items.
 
+### Key-Value Pair Extraction:
+  - `KeyValuePairExtractionScore`: computes key-wise metrics for each key in a JSON Schema-formatted key-value pair schema defining a set of keys to be extracted from an image, where each key-wise metric is inferred according to the key-value pair schema.
```

Comment:
probably need some more documentation here

Reply:
Great catch, thanks! Updated several other metrics and descriptions as well.